ui/src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react'
+import { Grid,Paper, TextField, Button, Typography,Link } from '@material-ui/core'
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Snackbar from '@material-ui/core/Snackbar';
+
+interface LoginProps {
+    isButtonDisabled: boolean;
+    setisButtonDisabled: (isButtonDisabled: boolean) => void;
+  }
+
+const initialState = {
+    username: '',
+    password: '',
+    
+  };
+
+  const validPassword = new RegExp('^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$');
+  const validEmail = new RegExp('^[a-zA-Z0-9._:$!%-]+@[a-zA-Z0-9.-]+.[a-zA-Z]$');
+
+  function isValidEmail(email : string) {
+    return validEmail.test(email);
+  }
+
+  function isValidPassword(password : string)
+  {
+    return validPassword.test(password);
+  }
+
+function Login(props: LoginProps){
+
+    const [formValues, setFormValues] = useState(initialState);
+    const [userNameError, setuserNameError] = useState<string | null>(null);
+    const [passwordError, setpasswordError] = useState<string | null>(null);
+    const [successMessage, setsuccessMessage] = useState<string | null>(null);
+    const [open, setOpen] = useState<boolean>(false);
+
+
+  const handleUserChange = (event: any) => {
+
+    if (!isValidEmail(event.target.value)) {
```

Comment:
Email and password validations should happen on submit instead of change otherwise validations will be triggered at every letter user types